lib/ansible/modules/monitoring/zabbix/zabbix_host_facts.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) me@mimiko.me
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+RETURN = '''
+---
+hosts:
+  description: List of Zabbix host. See https://www.zabbix.com/documentation/3.4/manual/api/reference/host/get for list of host values.
+  returned: success
+  type: dict
+  sample: [ { "available": "1", "description": "", "disable_until": "0", "error": "", "flags": "0", "groups": ["1"], "host": "Host A", ... } ]
+'''
+
+DOCUMENTATION = '''
+---
+module: zabbix_host_facts
+short_description: Gather facts about Zabbix host
+description:
+   - This module allows you to search for Zabbix host entries.
+version_added: "2.6"
+author:
+    - "(@redwhitemiko)"
+requirements:
+    - "python >= 2.6"
+    - zabbix-api
+options:
+    server_url:
```

Comment:
I would like to see a doc fragment implemented here just to prevent duplication:
https://github.com/ansible/ansible/blob/devel/lib/ansible/utils/module_docs_fragments/zabbix.py. 

```yaml
extends_documentation_fragment:
    - zabbix
```

Would be nice to have `validate_certs` option as well. See zabbix_host module for details on both.